packages/perspective-viewer-d3fc/src/less/chart.less

```diff
@@ -26,9 +26,6 @@
         & .x-label {
             height: 1.2em !important;
         }
-        & d3fc-canvas.plot-area {
```

Comment:
Rather than get rid of this, perhaps we could do:

````css
        & d3fc-canvas.plot-area {
            display: none;
        }
        &.d3_xy_scatter_canvas {
            & d3fc-canvas.plot-area {
                display: block;
            }
            & d3fc-svg.plot-area {
                display: none;
            }
        }
````

This should work, because we already ammend the class of the top-level chart element to include the chart's id.

It also means we can drop all the code where it sets the canvas display to "none"

Reply:
I'll get that changed